app/helpers/hyrax/collections_helper.rb

```diff
@@ -1,10 +1,29 @@
 # frozen_string_literal: true
 module Hyrax
   module CollectionsHelper
-    # TODO: we could move this to CollectionPresenter if it had a view_context
+    ##
+    # @since 3.0.0
+    # @return [#to_s]
+    def collection_metadata_label(collection, field)
+      Hyrax::PresenterRenderer.new(collection, self).label(field)
+    end
+
+    ##
+    # @since 3.0.0
+    # @return [#to_s]
+    def collection_metadata_value(collection, field)
+      Hyrax::PresenterRenderer.new(collection, self).value(field)
```

Comment:
i'm not sure if this "renderer" is actually helpful, over just having these helpers do the work directly:

  - it's not the same as other things we call `Renderer`
  - it says it's a `PresenterRenderer` but it only actually works for collection presenters (due to hard-coded partial resolution; though it's only *tested* for work presenters!).
  - most of the novel logic (now deprecated) is there just to yield itself.

i didn't want to mess with it too much in this PR, since my main goal is to move us toward being able to render Valkyrie collections (with a side goal of improving the story for collection metadata customization). this at least makes the whole thing invisible to the view, which can now just trust the helpers to do their thing.